packages/core/src/core/orchestrator.ts

```
@@ -398,89 +333,92 @@ export class Orchestrator {
+                            break;
+
+                        case HandlerRole.ACTION:
+                            const actionResult = await this.dispatchToAction(
                                 output.name,
-                                {
-                                    input: output.data,
-                                    result: actionResult,
-                                }
+                                request,
+                                output.data
                             );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)